libvirt/tests/src/snapshot/virsh_snapshot_stress.py

```
@@ -0,0 +1,200 @@
+    """Define a VM temporarily using XML string and validate it via virsh define."""
+    with tempfile.NamedTemporaryFile("w", delete=False) as f:
+        f.write(xml_str)
+        tmp = f.name
+    try:
+        res = virsh.define(tmp, debug=True, ignore_status=True)
+        _cmd_ok(test, res, f"virsh define {tmp}")
+    finally:
+        Path(tmp).unlink(missing_ok=True)
+
+
+def _retarget_disk_persistent_xml(test, vm_name, target_dev, new_path):
```

Ruff (0.13.3)

Unused function argument: `test`

(ARG001)